hub/util/dataset_utils.py

```diff
@@ -0,0 +1,35 @@
+from typing import Union
```

Comment:
can you rename this file to `hub/util/dataset.py`? `utils` is implicit in the parent module